### PR TITLE
F indigo devel port grasps 4

### PR DESCRIPTION
**Pull request by ugocupcic**
Adds sr_robot_msgs/GraspArray as it makes serialising lists of grasps much easier.
quick grasp takes a param for the file of grasps to use.
Adds a converter for the sr_hand style xml files.
Fixes a radians bug in new grasp ;-)
@ugo this now runs all the grasps from the interpolater but with the tactile stuff :)

### CODE REVIEW

File: sr_grasp/scripts/quick_grasp [hunk 1]
```diff
@@ -59,6 +60,15 @@ class QuickGraspNode(object):
         else:
             rospy.logerr("Fail!")
 
+    def new_grasp(self, name):
+        grasp = Grasp()
+        grasp.id = name
+        positions = self.hand.read_all_current_positions()
+        for k, v in positions.iteritems():
+            positions[k] = radians(v)
```
Comment (ugocupcic) [inline sr_grasp/scripts/quick_grasp:68]: so we store them in degrees in the yaml file (same as our old xml file). Probably ok as it's easier to read it, but wondering if it'd be cleaner to just have everything in radians directly. Thoughts?

**Comment by ugocupcic** [comment]
@markpitchless just one simple comment. Happy to merge and leave like that (or merge and add an issue to convert to rad depending on the result of our discussion).

File: sr_grasp/scripts/quick_grasp (hunk 1 above)
Comment (markpitchless) [inline sr_grasp/scripts/quick_grasp:68]: It is all in radians in the yaml! The shadow hand lib does degrees but that is a bad habit ;-)

Comment (ugocupcic) [inline sr_grasp/scripts/quick_grasp:68]: Ahah! right sorry. Can you add an issue, with a tag discussion or something in github to remember that?

Comment (markpitchless) [inline sr_grasp/scripts/quick_grasp:68]: #192

**Pull request merged**